src/form/PMForm/CustomFormResponse.php

```
@@ -0,0 +1,77 @@
+	public function __construct(array $data){
+		$this->data = $data;
+	}
+
+	public function getInt(string $name) : int{
+		$this->checkExists($name);
+		return $this->data[$name];
+	}
+
+	public function getString(string $name) : string{
+		$this->checkExists($name);
+		return $this->data[$name];
```

PHPStan (2.1.15)

Method pocketmine\form\PMForm\CustomFormResponse::getString() should return string but returns mixed.


(return.type)